test/testhelpers/helpers.go

```diff
@@ -39,6 +51,7 @@ func NewReport(name, namespace, testQueryName string, reportStart, reportEnd *ti
 		},
 		Spec: metering.ReportSpec{
 			QueryName:      testQueryName,
+			Inputs: inputs,
```

Comment:
Same comment as before. This is going to fail the verify check until you run gofmt locally and recommit.